### PR TITLE
Update IK Switch Styling

### PR DESCRIPTION
**Pull request by Geeoon**
Changed the cursor type to "not-allowed" when hovering over ik switch in states where it cannot be enabled.
Changed the theme of the switch to be even darker when it cannot be enabled.

### CODE REVIEW

**Review by abhaybd — DISMISSED** [review]
Looks like the compass changes got included in this pr, can you rebase as appropriate?

File: src/components/sidebar/ToggleInverseKinematics.css [hunk 1]
```diff
@@ -46,11 +46,16 @@
     height: 100%;
     border-radius: 100px;
     outline: 2px solid white;
+    box-sizing: border-box;
```
Comment (QuinnP2910) [inline src/components/sidebar/ToggleInverseKinematics.css:49]: This element has an outline, not a border, so changing box-sizing to border-box seems to have no effect.

**Review by QuinnP2910 — COMMENTED** [review]
Just one very small thing, otherwise looks good.

**Review by huttongrabiel — APPROVED** [review]
lgtm

**Pull request merged**